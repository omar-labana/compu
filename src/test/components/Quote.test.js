import React from 'react';
import '@testing-library/jest-dom';
import Quote from '../../components/Quote';
import { shallow } from 'enzyme';
import renderer from 'react-test-renderer';

const stl = () => 'red';
const doNothing = () => ''

describe('Quote Component', () => {
    let wrapper;
    wrapper = shallow(<Quote buttonName="1" st={stl} clickHandler={doNothing} />);

    it('exists', () => {
        expect(wrapper.find('Quote')).toBeTruthy();
    });

    it('has to match snap', () => {
        const quote = renderer.create(<Quote />);
        expect(quote).toMatchSnapshot();
    });
})