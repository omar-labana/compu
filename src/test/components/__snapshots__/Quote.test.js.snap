// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Quote Component has to match snap 1`] = `
<div>
  Mathematics is not about numbers, equations, computations, or algorithms: it is about understanding. â€“William Paul Thurston
</div>
`;
